backend/backend/errors/room.py

```diff
@@ -0,0 +1,32 @@
+from fastapi import status, Request
```

Comment:
You shouldn't import fastapi in BL

Reply:
so how do you suggest to create customs error?
should I just sprate them from the handlers than?